docs/podman-container.1.md

```diff
@@ -0,0 +1,41 @@
+% podman-container "1"
+
+## NAME
+podman\-container - Manage containers
+
+## SYNOPSIS
+**podman container SUBCOMMAND [OPTIONS]
```

Comment:
Don't they need to provide a CONTAINER too?

Reply:
That depends on the subcommand

podman container ls

Does not require a container.

Reply:
Gotcha, so it's buried away as part of the options.  Wish we could tease it out somehow on this page,  but I'm not thinking of a nice way to do that without making a huge mess.  Probably best off leaving it to the manpage links you've provided.